e2e/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "e2e-x-dash",
+  "version": "0.0.0",
+  "description": "This module enables you to write x-dash components that respond to events and change their own data.",
+  "keywords": [
+    "x-dash"
+  ],
+  "author": "",
+  "license": "ISC",
+  "scripts": {
+    "server": "node server.mjs",
+    "e2e": "webpack && jest -c ./jest.config.js --detectOpenHandles"
+  },
+  "dependencies": {
+    "@financial-times/x-engine": "file:../packages/x-engine",
+    "@financial-times/x-interaction": "file:../packages/x-interaction"
+  },
+  "devDependencies": {
+    "@financial-times/x-rollup": "file:../../packages/x-rollup",
+    "@financial-times/x-test-utils": "file:../../packages/x-test-utils",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "webpack-cli": "^4.8.0",
+    "puppeteer": "^10.4.0"
+  },
```

Comment:
**question**: how are these supposed to be installed? maybe this folder should be added to `workspaces` in the top-level `package.json` so Athloi can handle it?